docs/reference/tab-widgets/inference-api/infer-api-requirements.asciidoc

```diff
@@ -41,6 +41,13 @@ You can apply for access to Azure OpenAI by completing the form at https://aka.m
 
 // end::azure-ai-studio[]
 
+// tag::google-vertex-ai[]
+* A https://console.cloud.google.com/[Google Cloud account]
+* A project in Google Cloud
```

Comment:
Should we add the info that you have to enable Vertex AI API? Also should we mention the permission you need to have? We could also link to the corresponding pages mentioning the permissions you need, which probably makes it a bit more resilient to change on Google's side, but I don't know how detailed we wanna go here.